spec/controllers/code_owners/projects_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe CodeOwners::ProjectsController, type: :request do
```

Comment:
If this is a `request` spec, why name it projects_controller_spec.rb 🤔

Reply:
because as i think i understood, `type: :controller` is gonna be deprecated and should be changed by `type: : request`

Reply:
Actually, I think we don't need to pass this metadatum to `describe`, [as we already tell RSpec to infer its type](https://github.com/rootstrap/rs-code-review-metrics/blob/develop/spec/rails_helper.rb#L65).

You could move this file to `spec/requests` and rename it.